### DISCUSSION TITLE
$USER Constraint Value for Groups

### DISCUSSION THREAD

**Discussion by LukasIgel** [netbox-community/netbox#11938] [Q&A]
Dear community,

is there something similar to the $USER value for user groups for setting permissions?

I want to define a constraint in the sense of "created_by": "$GROUP", so that a user can only access an object which was created by a member of a group he belongs to. I guess that would mean that $GROUP would be an array, because a user can belong to more than one group, and I would have to ask that the object was created by at least one of the groups.

Thanks!
-Igel

**Comment by abhi1693** [comment]
This definitely does not exist yet but you can request it as an FR
